src/components/RichEditor/index.jsx

```diff
@@ -44,14 +45,24 @@ class RichEditor extends React.Component {
       input: {
         value,
       },
+      meta: {
+        touched,
+        error,
+      },
     } = this.props;
     const remainingChars = maxChars - this.state.charCount;
     const characterLimitMessage = `Recommended character limit (including spaces) is
       ${maxChars}. ${remainingChars} characters remaining.`;
 
     return (
       <React.Fragment>
-        <div id={id} className="form-group">{label}</div>
+        <div id={id} className="form-group mt-3">{label}</div>
```

Comment:
I might switch this to an `mb-2`? In testing, mt-3 doesn't do anything, it seems to overlap with the form above it's margin-bottom? But `form-group`'s default `mb` of 3 is too much compared to other fields. `mb-2` looks the most right on my screen anyway.

Reply:
With `mb-2`the major works editor gets pretty up close and personal with the bottom part of the bio editor, but if `mb-2` looks better to you I can switch it